runner/full_1min.go

```diff
@@ -51,33 +51,45 @@ func gatherQueryStatsForServer(ctx context.Context, server *state.Server, global
 		return newState, errors.Wrap(err, "error collecting query plan stats")
 	}
 
+	newState.ServerIoStats, err = postgres.GetPgStatIo(ctx, c, connection)
+	if err != nil {
+		return newState, errors.Wrap(err, "error collecting Postgres server statistics")
+	}
+
 	// Don't calculate any diffs on the first run (but still update the state)
 	if len(server.PrevState.StatementStats) == 0 || server.PrevState.LastStatementStatsAt.IsZero() {
 		return newState, nil
 	}
 
-	diffedStatementStats := diffStatements(newState.StatementStats, server.PrevState.StatementStats)
-	collectedIntervalSecs := uint32(newState.LastStatementStatsAt.Sub(server.PrevState.LastStatementStatsAt) / time.Second)
+	timeKey := state.PostgresStatementStatsTimeKey{
```

Comment:
I wonder if we should rename the `PostgresStatementStatsTimeKey` struct to `HistoricStatsTimeKey` or something, so its clear its not specific to statements?